src/org/pcic/util/Utils.groovy

```diff
@@ -0,0 +1,35 @@
+package org.pcic.util
+
+import org.pcic.GitUtils
+import org.pcic.DockerUtils
+
+
+class Utils implements Serializable {
+    def steps
+
+    Utils(steps) {
+        this.steps = steps
+    }
+
+    Map processParams(Map expected, Map args) {
+        def unknownKeys = (expected.keySet() + args.keySet()) - expected.keySet()
```

Comment:
If `+` is set union, and `-` is set difference, then this expression is equivalent to `args.keySet() - expected.keySet()`. Is that what you meant?

Reply:
Good catch, that's not supposed to be there.